client/extensions/woocommerce/app/dashboard/setup/store-location.js

```diff
@@ -290,10 +314,12 @@ function mapStateToProps( state, ownProps ) {
 	const settingsGeneralLoaded = areSettingsGeneralLoaded( state, siteId );
 	const storeLocation = getStoreLocation( state, siteId );
 	const locationsLoaded = areLocationsLoaded( state, siteId );
+	const countries = getAllCountries( state, siteId );
 	const countriesWithStates = getCountriesWithStates( state, siteId );
 
 	return {
 		contactDetails,
+		countries,
```

Comment:
Looks like this needs to be added to `propTypes`

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/22838/commits/705b284462b5390c6fa399f6c7eaef08f0cc96a9